lib/ClangImporter/ImportDecl.cpp

```diff
@@ -4113,6 +4134,30 @@ namespace {
             func->setImportAsStaticMember();
           }
         }
+
+        if (importedName.isSubscriptAccessor()) {
+          assert(func->getParameters()->size() == 1);
+          const auto parameterType = func->getParameters()->get(0)->getType();
+          if (!parameterType)
+            return nullptr;
+
+          auto cxxSubscripts = Impl.CXXSubscripts[parameterType];
```

Comment:
Just make this a reference type, then you don't need to update it later.

Reply:
Fixed